splunk-otel-android/src/main/java/com/splunk/rum/ErrorIdentifierExtractor.java

```diff
@@ -76,16 +77,29 @@ private String retrieveVersionCode() {
     }
 
     @Nullable
-    private String retrieveCustomUUID() {
+    private String retrieveSplunkBuildID() {
         if (applicationInfo == null) {
-            Log.e(SplunkRum.LOG_TAG, "ApplicationInfo is null; cannot retrieve Custom UUID.");
+            Log.e(SplunkRum.LOG_TAG, "ApplicationInfo is null; cannot retrieve Splunk Build ID.");
             return null;
         }
+
         Bundle bundle = applicationInfo.metaData;
-        if (bundle != null) {
-            return bundle.getString(SPLUNK_UUID_MANIFEST_KEY);
+        if (bundle == null) {
+            Log.d(SplunkRum.LOG_TAG, "Application MetaData bundle is null - no metadata present");
```

Comment:
nit: You have `MetaData` here, but few lines below you have `metadata`. I would unify to `metadata`.